posthog/api/user.py

```diff
@@ -455,6 +458,30 @@ def redirect_to_site(request):
     return redirect("{}#__posthog={}".format(app_url, state))
 
 
+@authenticate_secondarily
+def redirect_to_website(request):
+    team = request.user.team
+    app_url = request.GET.get("appUrl") or (team.app_urls and team.app_urls[0])
+
+    if not app_url:
+        return HttpResponse(status=404)
+
+    if not team or not urllib.parse.urlparse(app_url).hostname in ['localhost', 'posthog.com']:
+        return HttpResponse(f"Can only redirect to a permitted domain.", status=403)
+
+    token = jwt.encode({
+        "id": request.user.strapi_id,
+        "iat": int(time.time()),
+        "exp": int((datetime.now() + timedelta(days=30)).timestamp())
+    }, os.environ.get("JWT_SECRET_STRAPI"), algorithm='HS256')
+
+    # pass the empty string as the safe param so that `//` is encoded correctly.
+    # see https://github.com/PostHog/posthog/issues/9671
+    userData = urllib.parse.quote(json.dumps({ "jwt": token }), safe="")
+
+    return redirect("{}?userData={}&redirect={}".format('http://localhost:8001/auth', userData, app_url))
```

Comment:
I don't think the url here should be localhost.

Reply:
correct